index.html

```diff
@@ -82,10 +83,100 @@ <h2>Software Developer</h2>
         </li>
       </ul>
     </section>
-    <section class="vector display-flex justify-content">
-      <img src="Images/Vector-Icon.png" alt="A Vector Icon">
+    <section class="vector display-flex justify-content style">
       <div>
-        <h2 class="word">Work</h2>
+        <img src="Images/Vector-Icon.png" alt="A Vector Icon">
+      </div>
+      <div>
+        <h2 class="word style">Work</h2>
+      </div>
+    </section>
+  </section>
+  <section>
+    <header class="about-work display-flex">
+      <h1 class="style">MY RECENT<br> WORK</h1>
+    </header>
+  </section>
+  <div class="line2"></div>
+  <section class="background-div">
+    <section class="work-container">
+      <div class="cards-container display-flex">
+        <div class="placeholder"></div>
+        <header class="cards-header display-flex">
+          <h2 class="style">Multi-Post Stories</h2>
+        </header>
+        <div class="card-info">
+          <p class="style">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div class="languages-div">
+          <div class="css lang style">css</div>
+          <div class="html lang style">html</div>
+          <div class="bootstrap lang style">bootstrap</div>
+          <div class="ruby lang style">Ruby</div>
+        </div>
+        <div class="btn-div">
+          <button class="see-project">See Project</button>
+        </div>
+      </div>
```

Comment:
The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

 - [ ] Instead of using `div` tags for adding the project technologies you should use a list `ul` with `li` elements.
 - [ ] Please apply the same that I described above for the rest of your project technologies lists.

Reply:
Hi DanteAlonsoHT, my template does not have 'About Self section where I should use the list of project Technologies using ul and li tags.

Reply:
I know it, but it's a mandatory requirement for each project :)

![image](https://user-images.githubusercontent.com/60164239/129437526-bf2d8eaf-b544-4ca1-a1b6-0cc027cb4683.png)

Reply:
Do I go ahead and request another review, since my template does not have about section?

Reply:
It would be dangerous due to you have "My Recent Projects" section, and it's one of the requirements according to the project :c 
See again the image with the requirements:

![image](https://user-images.githubusercontent.com/60164239/129438099-2f4a5749-703c-4a74-b742-ce20fc696a29.png)

Reply:
Hi DanteAlonsoHT,

Kindly find below the link to the requirement for this project.
https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md

 Please, it says we should use the ul and li tags for the About Section which I do not have in my template.

![Requirement](https://user-images.githubusercontent.com/77106962/129438317-07eeae03-1624-40c8-8d2a-6add7de8f237.png)

Reply:
Sorry for my mistake, actually Microverse provided me other requirements for this project (Image I sent you before), but these requirements are missing in the requirements they provided you. You can go ahead with request another code review. 😉

Reply:
Ok, thank you for the approval. I appreciate the sincerity and professionalism.